src/ocean/util/container/more/Stack.d

```diff
@@ -351,43 +429,43 @@ static this ( )
 
 /*******************************************************************************
 
-*******************************************************************************/
+ *******************************************************************************/
 
 debug (Stack)
```

Comment:
Remove ?

Reply:
Already removed in 3.5.x, don't want conflicts

Reply:
But you changed the indent, so you'll get them anyway